concrete-core-fixture/src/fixture/glwe_ciphertext_ggsw_ciphertext_external_product.rs

```diff
@@ -30,6 +30,7 @@ pub struct GlweCiphertextGgswCiphertextExternalProductParameters {
     pub ggsw_noise: Variance,
     pub glwe_noise: Variance,
     pub glwe_dimension: GlweDimension,
+    pub ggsw_value_to_use: Option<usize>,
```

Comment:
encrypted_ggsw_value

Reply:
can it be Precision::Raw?

Reply:
We should just have it non-optional and set values (0 and 1) in the parameters iterator.